src/main/java/com/exasol/adapter/RequestDispatcher.java

```diff
@@ -40,10 +40,10 @@ public static String adapterCall(final ExaMetadata metadata, final String rawReq
 
     private static String processAdapterCall(final ExaMetadata metadata, final String rawRequest)
             throws AdapterException {
-        logVersionInformation();
-        logRawRequest(rawRequest);
         final AdapterRequest adapterRequest = parseRequest(rawRequest);
         configureAdapterLoggingAccordingToRequestSettings(adapterRequest);
```

Comment:
Hm, we have a bit of a chicken and egg problem. If parseRequest fails we don't get the raw request logged. Maybe we need separate the configuration of the remote log from parsing the request. However, I don't know what exactly parseRequest does.

Reply:
Ok, I had a look into parseRequest and actually contains quite a lot of logic. So, logging the rawRequest before it would be nice. However, we could also say we log the rawRequest first to console, such that you can get it via Alter session script_ourput_redirect or via the database log. The database logs have the disadvantage that users don't have easy access. So, logging to console is good extension here.